src/components/JdsSelect/Select.vue

```diff
@@ -129,16 +125,49 @@ export default {
      */
     valueKey: {
       type: String,
-      default: 'value',
+      default: "value",
     },
     /**
      * Required if options consist of objects.
      * Defined option label.
      */
     labelKey: {
       type: String,
-      default: 'label',
+      default: "label",
```

Comment:
**minor**
I checked on another component, there are still some use of single quote in this props default initiation section.
maybe need to be discussed in the team, whether to use single or double quotes for consistency of code.

Reply:
This will be fixed on another Pull Request, along with ESLint setup.